### DISCUSSION TITLE
更新系统，安装到另一个槽位后，超级用户列表需要重新授权吗

### DISCUSSION THREAD

**Discussion by popokcn** [tiann/KernelSU#2246] [Q&A]
每次更新完系统，安装全量包。重启以后，超级用户列表那里原来授权了root权限的软件都变umount了，需要手动从新一个一个授权root。

这是正常的吗，还是我操作有问题

**Comment by WHATHBER** [comment]
没遇到过这样的情况诶，是不是更新的时候把kernel SU APP数据给清理了啊

**Reply by popokcn** [reply]
没有清理ksu数据，酷安上好多人都有这个问题，他们说是ksu的bug，还说作者不打算修复

**Comment by qq2519157** [comment]
确实每次安装到未使用插槽后，需要重新授权

**Reply by popokcn** [reply]
有时候不是完全掉完，只是一部分掉授权，比如原来超级用户授权数量是10个，更新重启后，可能变成4个了，其中6个掉了。

**Comment by Fluair** [comment]
虽然我没遇到过，不过我想，如果能有备份设置和授权列表的功能就好了，即使授权列表丢失了也可以还原设置和授权列表
